lib/app/item.ex

```diff
@@ -85,6 +91,11 @@ defmodule App.Item do
     |> Repo.preload(tags: from(t in Tag, order_by: t.text))
   end
 
+  def get_draft_item(person_id) do
+    Repo.get_by(Item, status: 7, person_id: person_id) ||
+      Repo.insert!(%Item{person_id: person_id, status: 7})
```

Comment:
Create an item with the status id 7: https://github.com/dwyl/statuses/blob/d7a85d9cbd003c05d3371363d57f9d53d8e044c5/lib/statuses.ex#L45-L48

Reply:
Is it possible to have a small list of what each status number means in `buildit.md`?
This is because when I was making changes with the API, had some trouble finding the meaning (looked it here https://github.com/dwyl/mvp/blob/main/BUILDIT.md#item).
But it'd be nice to have a list for future reference if that's possible 😃

Reply:
I think the best file to look at for this information is https://github.com/dwyl/statuses/blob/d7a85d9cbd003c05d3371363d57f9d53d8e044c5/lib/statuses.ex

We could copy the id and meaning to the buildit.md but it will need to be synchronised each time the statuses file is changed.

Reply:
We 100% want to keep `statuses` as a separate repo so that we can use it in `Flutter`-land https://github.com/dwyl/statuses/issues/10